src/quicr_server_raw_session.cpp

```diff
@@ -107,14 +105,13 @@ QuicRServerRawSession::run()
 }
 
 void
-QuicRServerRawSession::publishIntentResponse(
-  const quicr::Namespace& quicr_namespace,
-  const PublishIntentResult& result)
+ServerRawSession::publishIntentResponse(const quicr::Namespace& quicr_namespace,
+                                        const PublishIntentResult& result)
 {
   if (!publish_namespaces.count(quicr_namespace))
     return;
 
-  // TODO: Need to update publish_namespaces and intent methods to support multi origin
+  // TODO: Support more than one publisher per ns
```

Comment:
This actually seems like something we should not support, to avoid sending of duplicate names.

Reply:
Leaving this unresolved and adding @TimEvens to the conversation.